### DISCUSSION TITLE
Image does not load when sharing link

### DISCUSSION THREAD

**Discussion by sandsoncosta** [hbstack/.github#89] [Issues and bugs]
Is there any reason why some links can load the post image and others can't?
![image](https://github.com/user-attachments/assets/057ad286-ac9c-43fa-acd0-f4b34ac496d5)
On WhatsApp, it only loads images when the title is small.
But on Teams, Telegram, LinkedIn, for example, it loads normally.
Is this a limitation of WhatsApp or does it have some logical basis?

**Comment by razonyang** [comment]
I don't think it's an issue of theme, perhaps there 's a limitation of WhatsApp.

> The theme will generate open graph meta by default, unless you've manually removed it.

- https://wordpress.org/support/topic/ogimage-not-showing-in-whatsapp/
- https://thospfuller.com/2023/12/24/open-graph-whatsapp/
- You can find more info about this by searching.

**Reply by sandsoncosta** [reply]
That's it!
![image](https://github.com/user-attachments/assets/4e48d143-bd30-4f0a-b098-5f553659ab44)
1734x1080 if i'm not mistaken.
I changed it to 1280x720
It works.
